### ISSUE TITLE
Skipping versions 5+?

### ISSUE THREAD

**Issue by kLabz** [kLabz/haxe-material-ui#21]
For more context:
* https://github.com/mui-org/material-ui/releases/tag/v5.0.0-rc.0
* https://github.com/mui-org/material-ui/discussions/27803

_Let's be honest, I didn't catch up to 5.0 alpha and beta versions, and don't have much time to do it these days, so this decision is also easier because of that xD_

tl;dr: material-ui is rebranding as "mui", which involves:
* Renaming many things, which can be a PITA for a haxe version supporting both <= 4 and >= 5
* Keeping material design as one of the possible design systems available
* _(TODO: more? don't have much time for that atm)_

What could be done, though, is create a separate "mui" haxe lib (by forking?), do the rebrand and updates there, and keep this library for material-ui (so <= 4)

**Comment by kevinresol** [comment]
I think I generally agree. Maintaining externs for upstream breaking versions is PITA. Let's encourage  people to start a haxe-mui or something if they need it.

**Comment by kLabz** [comment]
Indeed. I might even do it myself if I have the occasion. Or maybe dts2hx could work ?
I take it you're also using 4.x version ?

**Comment by kevinresol** [comment]
I just started a new repo here https://github.com/kevinresol/haxe-mui

Simple renaming of the npm packages in jsRequire meta seems working. But we will see.
